modules/userId/userId.md

```diff
@@ -169,7 +178,17 @@ pbjs.setConfig({
                  type: 'html5',
                  name: '_criteoId',
                  expires: 1
-        }
+                 }
+     },{
+            name: "originFloc",
+            params: {
+                token: "SHAREDID_TOKEN"
+                },
+            storage: {
+            type: "html5",
```

Comment:
dont store the floc, get it fresh every time

Reply:
if this is the approach, then all examples should remove the `storage` config object. even better, a warning/error should be thrown in the `getId` function if `storage` was set at all, forcing publishers NOT to set it...but that depends on how important it is to fetch a fresh value every time or not

Reply:
Excellent point

Reply:
@icflournoy you may want to do the same thing,  as storage interferes with your functionality and RDM sets it

Reply:
@patmmccann @smenzer Have updated to get cohort id from browser and removed the storage.